sdk/include/ndk/kefuncs.h

```diff
@@ -73,6 +73,13 @@ KiDeliverApc(
 //
 // Process/Thread Functions
 //
+CODE_SEG("INIT")
+VOID
+NTAPI
+KeStartAllProcessors(
```

Comment:
This is an function internal to ntoskrnl and is not an export --> it should not be placed in this file, but instead in ntoskrnl/includes/internal/ke.h